src/features/textEditor/components/TipTapEditor.tsx

```diff
@@ -77,7 +75,7 @@ export function TipTapEditor({ editorContent, isActive, saveFileInMemory, update
   return (
     <div className="flex h-full w-full flex-col" onClick={focusEditorOnClick}>
       <MenuBar editor={editor} />
-      <EditorContent className="tiptap-editor" editor={editor} />
+      <EditorContent autoFocus className="tiptap-editor" editor={editor} />
```

Comment:
Could you remove `autoFocus`, since this prop is not working. So we know that the autofocus actually comes from the `useEffect`